server/app/forms/admin/BulkStatusUpdateForm.java

```diff
@@ -5,38 +5,37 @@
 
 /** A form for selecting multiple applications to be considered for a status update. */
 public final class BulkStatusUpdateForm {
-  private List<String> applicationsIds;
+  private List<Long> applicationsIds;
   private String statusText;
-  private Boolean maybeSendEmail;
+  private Boolean shouldSendEmail;
 
-  public Boolean isMaybeSendEmail() {
-    return maybeSendEmail;
+  public Boolean sendEmail() {
```

Comment:
Suggestion/Nit: If the public-facing method is `sendEmail()`, should the private variable be just `sendEmail` as well? Or, should the public-facing methods be named `getShouldSendEmail()` and `setShouldSendEmail(...)` for consistency?